src/foraging_gui/Foraging.py

```diff
@@ -128,11 +128,16 @@ def __init__(self, parent=None,box_number=1,start_bonsai_ide=True):
         self.threadpool6=QThreadPool() # for saving data
         self.threadpool_workertimer=QThreadPool() # for timing
 
-        # create bias indicator
+        # create bias indicator and initialize parameters
         self.bias_n_size = 500
+        self.bias_step_size_mm = .2    # TODO: What value to set this at initially? Where to store?
```

Comment:
I think hard-coding 200um is good for now. It could be added as a curriculum parameter